sample/src/main/kotlin/com/chuckerteam/chucker/sample/GqlHttpTask.kt

```diff
@@ -0,0 +1,56 @@
+package com.chuckerteam.chucker.sample
+
+import okhttp3.OkHttpClient
+import retrofit2.*
+import retrofit2.converter.gson.GsonConverterFactory
+import retrofit2.http.Body
+import retrofit2.http.Headers
+import retrofit2.http.POST
+
+class GqlHttpTask(
+    client: OkHttpClient,
+) : HttpTask {
+
+    private val api = Retrofit.Builder()
```

Comment:
I think that the vast majority of people will use GraphQL with something like Apollo. I would prefer to see a real-life scenario integration within our sample.